addons/resource/models/resource_mixin.py

```diff
@@ -89,19 +89,20 @@ def _get_work_days_data_batch(self, from_datetime, to_datetime, compute_leaves=T
             mapped_resources[calendar or record.resource_calendar_id] |= record.resource_id
 
         for calendar, calendar_resources in mapped_resources.items():
-            day_total = calendar._get_resources_day_total(from_datetime, to_datetime, calendar_resources)
+            if calendar:
```

Comment:
invert condition to avoid code indent.
```suggestion
            if not calendar:
                continue
```

Reply:
done